### PR TITLE
ops/image face blur mapper

### PR DESCRIPTION
**Pull request by drcege**
Image OPs
* add `image_face_blur_mapper`
* rename `face_area_filter` to `image_face_ratio_filter`

Video OPs
* will add `video_face_blur_mapper` in another PR
* is it necessary to add `video_face_ratio_filter`?  @yxdyc  @HYLcool

### CODE REVIEW

File: docs/Operators.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The operators in Data-Juicer are categorized into 5 types.
 | Type                              | Number | Description                                     |
 |-----------------------------------|:------:|-------------------------------------------------|
 | [ Formatter ]( #formatter )       |   7    | Discovers, loads, and canonicalizes source data |
-| [ Mapper ]( #mapper )             |   38   | Edits and transforms samples                    |
+| [ Mapper ]( #mapper )             |   39   | Edits and transforms samples                    |
```
Comment (HYLcool) [inline docs/Operators.md:14]: This number needs to be updated after merging the latest main branch into this branch. Same in the ZH docs.

File: tests/ops/mapper/test_image_face_blur_mapper.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        ds_list = [{
+            'images': [self.img2_path]
+        }, {
+            'images': [self.img3_path]
+        }, {
+            'images': [self.img1_path]
+        }]
+        op = ImageFaceBlurMapper(blur_type='mean')
+        self._run_helper(op, ds_list)
+
```
Comment (HYLcool) [inline tests/ops/mapper/test_image_face_blur_mapper.py:95]: Do we need to test the op in the parallel mode (np > 1)?

**Review by HYLcool — COMMENTED** [review]
Besides, we need to add this new op to the sphinx docs like this [PR](https://github.com/alibaba/data-juicer/pull/236#discussion_r1520940537) does.

**Review by HYLcool — APPROVED** [review]
LGTM

**Comment by drcege** [comment]
> Besides, we need to add this new op to the sphinx docs like this [PR](https://github.com/alibaba/data-juicer/pull/236#discussion_r1520940537) does.

added

**Pull request merged**